### PR TITLE
Added virtualization to display visible part of Scripture text

### PR DESCRIPTION
**Pull request by tjcouch-sil**

### CODE REVIEW

File: react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx [hunk 1]
```diff
@@ -490,32 +455,21 @@ const withScrMarkers = (editor: CustomEditor): CustomEditor => {
...
-                                console.log(editor.selection);
                             }
                         } else {
                             // Closing an existing marker
                             // Get closest element of this marker style
                             const markerElement = Editor.above(editor, {
                                 match: (n) =>
                                     Element.isElement(n) &&
-                                    n.type !== 'editor' &&
+                                    // n.type !== 'editor' &&
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx:465]: Super minor thing. Can this be removed? You might need it later so do whatever you think is best.

**Review by katherinejensen00 — APPROVED** [review]
Looks great. Way to go on figuring out virtualization!

File: react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx (hunk 1 above)
Comment (tjcouch-sil) [reply react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx:465]: Good call - I will remove it. Thank you!

File: react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx [hunk 2]
```diff
@@ -881,12 +835,421 @@ export const ScriptureTextPanelSlate = ScriptureTextPanelHOC(
...
+                <div
+                    id={getScriptureChunkEditorSlateId(
+                        editorGuid,
+                        chapter,
+                        virtualizedIndex,
+                    )}
+                >
+                    {/* -------------------
+                    {`${
+                        scrChapterChunk.chunkNum
```
[the remark this answers is not in the document]
Comment (tjcouch-sil) [reply react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx:851]: Yep, I should remove this. Thank you!

**Review by katherinejensen00 — APPROVED** [review]

**Pull request merged**